src/test/ui/parser/increment-notfixed.rs

```diff
@@ -1,3 +1,5 @@
+// run-rustfix
```

Comment:
Given this, we might want to change the name (or merge it with the likely existing test that already had rustfix enabled).